keg_elements/db/mixins.py

```diff
@@ -67,30 +67,116 @@ def from_dict(self, data):
             else:
                 setattr(self, key, value)
 
-    def to_dict(self, exclude=None):
-        if exclude is None:
-            exclude = []
-        insp = sainsp(self)
-        return dict((attr.key, attr.value)
-                    for attr in insp.attrs if attr.key not in exclude)
+    def to_dict(self, exclude=[], hybrids=[]):
```

Comment:
Nice find @bladams.  Those should have never been lists due to the "accidentally mutated a default argument" possibility.